### ISSUE TITLE
package as a cocoapod?

### ISSUE THREAD

**Issue by ericcj** [mweisman/ShapeKit#4]
would love to use this if it was on cocoapods.org

**Comment by andreacremaschi** [comment]
Yeah, it would be great. I use cocoapods too, but I don't have any experience with pod creation. 
Furthermore, since ShapeKit relies on GEOS frameworks, it would be to investigate on how to redistribute them without infringing LGPL license. Now this is addressed (as it is upstream) with a script that download and build the library as a static one.

**Comment by scspijker** [comment]
For future reference: 

https://github.com/andreacremaschi/ShapeKit
https://github.com/Ancestry/CocoaPod-Specs/tree/master/ShapeKit
